index.html

```diff
@@ -1,119 +1,206 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!--Import Google Icon Font-->
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <!-- Compiled and minified CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/css/materialize.min.css"
+    />
     <link rel="stylesheet" href="./assets/css/style.css" />
     <title>Lars Euler</title>
-</head>
+  </head>
 
-<body>
+  <body>
     <!--Header including navigation bar-->
     <header class="header">
-        <h1 id="myname">Lars Euler</h1>
+      <h1 id="myname">Lars Euler</h1>
     </header>
 
     <!--Navigation Bar-->
-    <nav class="navigation">
-        <ul>
-            <li><a href="#detailsaboutme">About Me</a></li>
-            <li><a href="#surfreport">Work</a></li>
-            <li><a href="#contact-me">Contact Me</a></li>
+    <!-- Navbar medium up -->
+    <nav>
+      <div class="nav-wrapper teal darken-4">
+        <a href="#" data-target="mobile-demo" class="sidenav-trigger right"
+          ><i class="material-icons">menu</i></a
+        >
+        <ul class="right hide-on-med-and-down">
+          <li>
+            <a href="#detailsaboutme"
```

Comment:
Consider using kebab-cases for your id names as this seems to be more common in the workplace